### PR TITLE
Revision/update phenomic 0.14.0

### LINKED ISSUE #220: Script build broken
Any ideas? Using node v6.4.0 and npm 3.10.6

```
module.js:457
    throw err;
    ^

Error: Cannot find module '/Applications/postcss.org/Applications/postcss.org/node_modules/.bin/phenomic'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Applications/postcss.org/node_modules/phenomic/lib/bin/runner-cmd.js:12:1)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

@thangngoc89 ?

### PR DESCRIPTION
**Pull request by halvves**
Fixes #220 and is one step closer to #221 

I've updated to Phenomic 0.14.0 and updated the project structure to best (with my limited knowledge) reflect structure for 0.14.0.

The project now builds/runs properly at the very least.

I have started working to get upgraded to Phenomic 0.16.2, but am running into a few problems. I will go into depth on that issue.

### CODE REVIEW

**Comment by halvves** [comment]
I'm still experiencing one error, coming from highlight.js.

```
/Users/halvves/dev/opensource/postcss.org/node_modules/highlight.js/styles/tomorrow.css:4
.hljs-comment {
^
SyntaxError: Unexpected token .
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Module._extensions..js (module.js:565:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/halvves/dev/opensource/postcss.org/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at eval (eval at <anonymous> (/Users/halvves/dev/opensource/postcss.org/node_modules/.cache/phenomic/phenomic.node.bundle.js:5:29610), <anonymous>:1:18)
```

**Comment by halvves** [comment]
Just some additional information. Running in development mode (`npm start`) this error doesn't appear and the site spins up fine.

This error only occurs during `phenomic build` (`npm run build`). I can get phenomic build to run fine if I comment out [line 3 of ./web_modules/InANutshell/index.js](https://github.com/postcss/postcss.org/blob/master/web_modules/InANutshell/index.js#L3) (importing the highlightjs theme).

**Comment by MoOx** [comment]
You probably need to adjust a loader scope (include/exlude pattern) to be able to consume this file as a global css thing.

File: webpack.config.babel.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                ? "[hash:base64:5]"
+                : "[path][name]--[local]--[hash:base64:5]"
+              }`,
+              "postcss-loader",
+            ].join("!"),
+          ),
+        },
+        {
+          test: /\.css$/,
+          include: path.resolve(__dirname, "node_modules"),
```
Comment (MoOx) [inline webpack.config.babel.js]: This looks correct, not sure why you get an issue with highlight css

**Comment by marcustisater** [comment]
@halvves mind looking into it? Otherwise I will try make time for it.

**Pull request merged**